<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Realizar Orçamento - Cliente</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>741</width>
      <height>161</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Código</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descrição Produto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unidade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Qtd</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Preço Lista</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Desconto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Preço Unidade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Preço Total</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Frete</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_criar">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>230</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar Orçamento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_visualizar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Atualizar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_desc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="linha1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrição Produto</string>
    </property>
   </widget>
   <widget class="QLabel" name="linha2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unidade</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_uni">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_qtd">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="linha3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Qtd</string>
    </property>
   </widget>
   <widget class="QLabel" name="linha4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Preço Lista</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lista">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="linha5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desconto</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_desconto">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="linha6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Preço Unidade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_unid">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="linha7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Preço Total</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_total">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="linha8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_frete">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calcular">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular Orçamento</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Visualizar Orçamentos - Clientes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
